public/style.css

```diff
@@ -1,7 +1,6 @@
 html,
 body{
   font-family: "Sen", sans-serif;
-  overflow-x: hidden;
```

Comment:
This was causing some weird issues where two scroll bars were appearing on screen.